monokai-theme.el

```diff
@@ -5652,6 +5652,44 @@ Also affects 'linum-mode' background."
      ((,class (:foreground ,monokai-green))
       (,terminal-class (:foreground ,terminal-monokai-green))))
 
+   ;; which-key
+   `(which-key-key-face
+     ((,class (:foreground ,monokai-green
+			   :weight bold))
+      (,terminal-class (:foreground ,terminal-monokai-green
+				    :weight bold))))
+
+   `(which-key-separator-face
+     ((,class (:foreground ,monokai-comments))
+      (,terminal-class (:foreground ,terminal-monokai-comments))))
+
+   `(which-key-note-face
+     ((,class (:foreground ,monokai-comments))
+      (,terminal-class (:foreground ,terminal-monokai-comments))))
+
+   `(which-key-command-description-face
+     ((,class (:foreground ,monokai-fg))
+      (,terminal-class (:foreground ,terminal-monokai-fg))))
+
+   `(which-key-local-map-description-face
+     ((,class (:foreground ,monokai-yellow-hc))
+      (,terminal-class (:foreground ,terminal-monokai-yellow-hc))))
+
+   ;; `(which-key-highlighted-command-face
+   ;;   ((,class (:foreground ,monokai-green))
+   ;;    (,terminal-class (:foreground ,terminal-monokai-green))))
+
+   `(which-key-group-description-face
+     ((,class (:foreground ,monokai-red
+			   :weight bold))
+      (,terminal-class (:foreground ,terminal-monokai-red
+				    :weight bold))))
+
+   ;; `(which-key-special-key-face
```

Comment:
https://github.com/justbur/emacs-which-key/blob/master/which-key.el#L405

It looks like it inherits the key-face but inverts it. The face is applied under these conditions: ""These keys will automatically be truncated to one character
and have `which-key-special-key-face' applied to them. This is
disabled by default. Try this to see the effect."